From af5c5d93b54c957bbd3f16e9cbe710e3bd7c857f Mon Sep 17 00:00:00 2001
From: Lionel Debieve <lionel.debieve@foss.st.com>
Date: Wed, 23 Feb 2022 14:12:36 +0100
Subject: [PATCH 392/529] ARM: dts: stm32: enable crypto accelerator on
 stm32mp1 boards

Add the crypto accelerator enable on all boards that have the
crypto peripheral available.

Signed-off-by: Lionel Debieve <lionel.debieve@foss.st.com>
Change-Id: Ica34c8f10988a95e0c71228d62ca2a0e0802a60e
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/241001
Reviewed-by: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/280580
Tested-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Reviewed-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Domain-Review: Eric FOURMONT <eric.fourmont-ext@st.com>
---
 arch/arm/boot/dts/stm32mp135f-dk.dts | 4 ++++
 1 file changed, 4 insertions(+)

--- a/arch/arm/boot/dts/stm32mp135f-dk.dts
+++ b/arch/arm/boot/dts/stm32mp135f-dk.dts
@@ -185,6 +185,10 @@
 	status = "okay";
 };
 
+&cryp {
+	status = "okay";
+};
+
 &dcmipp {
 	status = "okay";
 	pinctrl-names = "default", "sleep";
